MvvmGoodWeatherApp/ViewModels/WeatherListViewModel.swift

```diff
@@ -8,10 +8,10 @@
 
 import Foundation
 
-struct WeatherListViewModel {
-    private var weatherViewModels = [WeatherViewModel]()
+class WeatherListViewModel {
```

Comment:
Refactoring struct to class for the reference type binding details view controller